frontend/src/metabase/ui/components/feedback/Skeleton/Skeleton.tsx

```diff
@@ -1,3 +1,19 @@
-export type { SkeletonProps } from "@mantine/core";
-export { Skeleton } from "@mantine/core";
-export { getSkeletonOverrides } from "./Skeleton.styled";
+import type { SkeletonProps } from "@mantine/core";
+import { Skeleton as MantineSkeleton } from "@mantine/core";
+import { useMemo } from "react";
+
+export const Skeleton = ({
```

Comment:
(i don't know the answer to this question)
do we need to add storybook stories for new ui components?

Reply:
Yeah the protocol is generally to add an entry to Storybook if you're adding a new export from `metabase/ui` to help with component discovery and reuse.